src/pages/search/advanced_search/AdvancedSearchPage.tsx

```diff
@@ -97,8 +97,10 @@ export const AdvancedSearchPage = () => {
           </Grid>
 
           <Grid container direction={isLargeScreen ? 'row' : 'column'} gap={2} size={12}>
-            <Grid sx={{ width: 'fit-content' }}>
-              <StyledFilterHeading>{t('search.advanced_search.publishing_period')}</StyledFilterHeading>
+            <Grid component="fieldset" sx={{ width: 'fit-content', border: 'none' }}>
```

Comment:
Denne har fått ekstra padding nå. legge til `padding: 0` også for fieldset i theme?

![bilde](https://github.com/user-attachments/assets/f684fe16-25ac-4701-be33-3a402db1783a)

Reply:
Ser ut som det kommer en bitteliten margin også, så den må nok også nullstilles